clusters.json

```diff
@@ -0,0 +1,41 @@
+{
+    "uan*": {
+        "name": "LUMI",
+        "vars": {
+            "HF_HOME": "/pfs/lustrep4/projappl/project_462000963/evals/hf_data",
+            "OUTPUT_DIR": "/pfs/lustrep4/projappl/project_462000963/evals/${USER}",
+            "VENV_DIR": "/pfs/lustrep4/projappl/project_462000963/evals/.venv",
+            "UV_LINK_MODE": "copy",
```

Comment:
Wonder whether having an inheritance mechanism would be better than copy-pasting bunch of stuff, no need for this version though.

We could also have a "common" set of env variables which is run first before the cluster specific.